.gitignore

```diff
@@ -44,3 +44,4 @@ Thumbs.db
 
 .nx/cache
 .angular
+.idx/dev.nix
```

Comment:
?

Reply:
IDX.dev Development workspace from Google. Preparing pr to save default configuration into repo. Will be ready little bit later. I use it to work on this project